apps/ensadmin/src/components/recent-registrations/types.ts

```diff
@@ -15,12 +15,59 @@ export interface Domain {
   };
 }
 
+/**
+ Data associated with a Registration event.
+ **/
+export interface LatestRegistration {
+  /**
+   * recently registered ENS name
+   */
+  name: string;
+
+  /**
+   * a UNIX timestamp in seconds of when the name was originally created
+   */
+  createdAt: number;
+
+  /**
+   * a UNIX timestamp in seconds of when will the current owner's tenure end
+   */
+  expiryDate: number;
```

Comment:
```suggestion
  expiresAt: number;
```

It's confusing to call this a "date" as that could be interpreted as just a date ex: "Jan 1, 2025" rather than a full timestamp.